QuestionBot/QuestionBot.UnitTests/Model/InMemoryStoreTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using QuestionBot.Model;
+
+namespace QuestionBot.UnitTests.Model {
+
+    [TestFixture]
+    public class InMemoryStoreTests{
+        private IStore _storeTest;
+        private IEnumerable<IRecord> _retrievedRecords = new List<IRecord>();
+
+        [SetUp]
+        public void Setup(){
+            _storeTest = new InMemoryStore();
+        }
+
+        [Test]
+        public void New_question_creates_record(){
+            string question = "What is 1+2?";
+
+            IRecord questionRecord = _storeTest.CreateRecord(question);
+
+            Assert.AreEqual(questionRecord.Question, question);
+            Assert.AreNotEqual(questionRecord.TimeAsked, DateTime.MinValue);
```

Comment:
I briefly mentioned this before, but all the `Assert.AreNotEqual` lines should be comparing to `default(DateTime)` or `default(int)` (depending on the type) to make them more clear what the assert is doing The other option would be to just delete them, trading a minuscule amount of "security" (that the module is doing what it's supposed to do) for a large amount of readability (implying that the `Assert.AreNotEqual` lines make the tests hard to understand).